src/applications/SRAnonTool/CTP_DicomToText.py

```diff
@@ -49,15 +49,19 @@
 # PatientID mapping from CHI to EUPI
 
 def patientid_map(PatientID):
-    eupi = IdentifierMapper.CHItoEUPI().lookup(PatientID)
+    try:
+        eupi = IdentifierMapper.CHItoEUPI().lookup(PatientID)
+    except Exception as e:
+        #print(e)
+        eupi = None
     if not eupi:
         return 'UNKNOWN'
     return eupi
 
 
 # ---------------------------------------------------------------------
 
-def extract_mongojson(mongojson, output, metadata_output=None):
+def extract_mongojson(mongojson, output, metadata_output=None, DicomTextArgs = {}):
```

Comment:
I think this will cause a subtle bug, because Python is silly - https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments